common/src/main/java/com/microsoft/identity/common/internal/authorities/AADTestAuthority.java

```diff
@@ -0,0 +1,59 @@
+package com.microsoft.identity.common.internal.authorities;
+
+import com.microsoft.identity.common.internal.logging.Logger;
+import com.microsoft.identity.common.internal.providers.microsoft.microsoftsts.MicrosoftStsOAuth2Configuration;
+import com.microsoft.identity.common.internal.providers.oauth2.OAuth2Strategy;
+import com.microsoft.identity.common.internal.providers.ropc.ResourceOwnerPasswordCredentialsTestStrategy;
+
+import java.util.Map;
+
+public class AADTestAuthority extends AzureActiveDirectoryAuthority {
+
+    private static transient final String TAG = AADTestAuthority.class.getSimpleName();
```

Comment:
Curious why `transient` for a tag?

Reply:
Probably copied that line from AzureActiveDirectoryAuthority as that uses transient. I will remove it. I believe it also needs to be removed from AzureActiveDirectoryAuthority